src/examples/using-with-cmake/host-config.cmake.in

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
```

Comment:
Note: These new auto-generated host-configs for the examples capture information necessary to build the examples, like the compiler and paths to TPLs.